finality-provider/service/fp_instance.go

```diff
@@ -685,9 +675,8 @@ func (fp *FinalityProviderInstance) getPollerStartingHeight() (uint64, error) {
 		startHeight = fppath.MaxUint64(startHeight, latestFinalisedBlocks[0].Height+1)
 	}
 
-	// consider the height after the last processed block
-	lastProcessedHeight := fp.GetLastProcessedHeight()
-	startHeight = fppath.MaxUint64(startHeight, lastProcessedHeight+1)
+	// consider the height after the last voted height
+	startHeight = fppath.MaxUint64(startHeight, fp.GetLastVotedHeight()+1)
```

Comment:
```suggestion
	startHeight = max[uint64](startHeight, fp.GetLastVotedHeight()+1)
```

Reply:
Nit, let's remove this util function and use go's built-in one. I see that we always pass 2 arguments anyways

Reply:
Good point!